resources/lang/de.js

```diff
@@ -48,6 +48,9 @@ class CactbotLanguageDe extends CactbotLanguage {
       Sludge: 'Schlamm',
       Doom: 'Verhängnis',
       StoneCurse: 'Steinfluch',
+      Imp: 'Flusskobold',
+      Toad: 'Frosch',
+      FoolsTumble: 'Trügerischer Sturz', // 0x387
```

Comment:
Sorry, but if you're going to leave a comment on the id, it's not 0x387 (which is hex).  It's either 387 or 0x183.  (or no need for comment at all).

Reply:
Oof. Counting is hard, you're right.